GOVERNANCE.md

```diff
@@ -0,0 +1,62 @@
+# Governance
+
+Meet the current members of the Node.js userland migrations:
+
+- [@AugustinMauroy](https://github.com/AugustinMauroy) - **Augustin Mauroy** (he/him)
+- [@avivkeller](https://github.com/avivkeller) - **Aviv Keller** (he/him)
+- [@bmuenzenmeyer](https://github.com/bmuenzenmeyer) - **Brian Muenzenmeyer** (he/him)
+- [@JakobJingleheimer](https://github.com/JakobJingleheimer) - **Jacob Smith** (he/him) - Maintainer
+- [@ljharb](https://github.com/ljharb) - **Jordan Harband** (he/him)
+
+## Rights
+
+Any team member can block a pull request or decision. A block can be overturned by a team vote, which requires a quorum of two-thirds of active members and passes with a simple majority (>50%). Pursuant to Node.js policy, the [TSC](https://github.com/nodejs/TSC) may override a team’s vote.
+Any member can create a tag on the repository's `HEAD` branch, which triggers a CI run and initiates a release. Tags must follow the format `vX.Y.Z@workspace`, where `X.Y.Z` is the version number and `workspace` is the name of the npm workspace (e.g. `v1.2.3@codemod`).
```

Comment:
I think we said this would be reserved for maintainers

Reply:
Okay. But is it setup now ?

Reply:
No, currently all team members have `maintain` permissions. Once this is finalised, I'll update the permissions inherited via team status (to `write`) and then assign specific people `maintain`.

Reply:
Okay ! got it